pkgs/top-level/stage.nix

```diff
@@ -272,8 +272,10 @@ let
           if stdenv.isLinux
           then makeMuslParsedPlatform stdenv.hostPlatform.parsed
           else stdenv.hostPlatform.parsed;
-      } // lib.optionalAttrs (stdenv.hostPlatform.system == "powerpc64-linux") {
-        gcc.abi = "elfv2";
+        gcc = (lib.optionalAttrs (stdenv.hostPlatform ? gcc) stdenv.hostPlatform.gcc) //
+          lib.optionalAttrs (stdenv.hostPlatform.system == "powerpc64-linux") {
+            abi = "elfv2";
```

Comment:
Not sure whether it makes sense to override this, even if it's been explicitly set.

Reply:
Maybe only override it if not set in `stdenv.hostPlatform.gcc`?

Reply:
Done